### ISSUE TITLE
Checkout: Cart icon has no focus state, can't select checkout via keyboard

### ISSUE THREAD

**Issue by ryelle** [Automattic/wp-calypso#21659]
Testing with VoiceOver + Safari, and keyboard-only + Chrome. If I have an item in my cart, I can't get to checkout using just my keyboard.

**To replicate:**

- Add something to your cart, then back out of checkout
- Visit `/plans/:site`, see the pulsing cart icon
- Try to navigate to the cart using your keyboard. You'll see focus on My Plan, Plans, and then the cart is focused, but there is no indicator:

<img width="109" alt="screen shot 2018-01-18 at 1 09 45 pm" src="https://user-images.githubusercontent.com/541093/35113854-ecdc6d92-fc50-11e7-8d0a-47c5de068c98.png">

- Hit enter to open the cart popover
- Try to tab to the checkout button
- You can't - not without tabbing through the entire page and sidebar, which is not expected or intuitive.
